pilot/pkg/networking/plugin/mixer/mixer.go

```diff
@@ -48,6 +48,16 @@ const (
 
 	// defaultConfig is the default service config (that does not correspond to an actual service)
 	defaultConfig = "default"
+
+	// policyCheck is the node label holding the policy check option override
+	policyCheck = "POLICY_CHECK"
+)
+
+// Policy check options control how Mixer checks are applied on per-pod basis
+const (
+	PolicyCheckEnable      string = "enable"
+	PolicyCheckDisable            = "disable"
+	PolicyCheckEnableAllow        = "allow-on-error"
```

Comment:
I think users are already using 'fail open'/'fail close' - 'allow-on-error' may not be most intuitive.

Reply:
the issue is that anyone new to Istio will be confused by our inverse use of open and close here (compared to say circuit breaking). heck, our own team has been confused by this before.

`allow-on-error` isn't elegant, perhaps, but it at least attempts at improving clarity.

open to better suggestions / thoughts, however.